core/dbt/main.py

```diff
@@ -142,7 +142,6 @@ def main(args=None):
             exit_code = e.code
 
         except BaseException as e:
-            traceback.print_exc()
```

Comment:
Given that this was an incidental inclusion in v1.2.0, I'd also be in favor of a PR that removed this line alone, and backporting that PR to `1.2.latest` for inclusion in a v1.2.1 patch release

Reply:
I'm actually looking at two different problems, really, the lack of a stack trace for top level exceptions and catching exceptions in threads. I could make the current code the "lack of a stack trace" fix and backport it?

Reply:
I like that idea!

Reply:
good plan.